sessionctx/variable/session.go

```diff
@@ -787,6 +787,9 @@ type SessionVars struct {
 	// DurationWaitTS is the duration of waiting for a snapshot TS
 	DurationWaitTS time.Duration
 
+	// WaitLockLease is the duration of cached table read lease expiration time.
+	WaitLockLeaseTime time.Duration
```

Comment:
Can we put it some place else or direct add on the execdetil wait time?
Introduce this variable in `SessionVars` is ugly